.github/workflows/cd.yml

```diff
@@ -1,18 +1,10 @@
 name: cd
 on:
-  workflow_run:
-    workflows: ["ci"]
-    branches-ignore: ["*"]
-    types:
-      - completed
   push:
     tags:
       - "v*"
 
-permissions:
-  contents: read
-
 jobs:
   plugin-cd:
-    uses: mattermost/actions-workflows/.github/workflows/plugin-cd.yml@main
+    uses: mattermost/actions-workflows/.github/workflows/community-plugin-cd.yml@main
```

Comment:
I think we should fixate the version instead of using `main`. We tend to break CI from time to time, which is fine for the plugins we maintain, but for community-maintained ones, it adds unnecessary frustration.

Reply:
That is the reason we are going to use a different workflow than the one we are using . Community can use this one created by us or not . It's their choice . 
We will not mess with the `community-plugin-cd` and `community-plugin-ci` workflows . That's the idea

Reply:
If we are not going to update the community ci/cd workflows, there is no harm with pinning the version, right? :wink: 

> We will not mess with the `community-plugin-cd` and `community-plugin-ci` workflows . That's the idea

I don't think that we will never update these two workflows. There will be cases where we want to update a plugin and I really want to avoid breaking CI on other ones.